tests/backup/backup_partial_test.go

```diff
@@ -471,3 +474,323 @@ var _ = Describe("{PartialBackupSuccessWithPxVolumes}", func() {
 
 	})
 })
+
+var _ = Describe("{DifferentBackupStatusValidation}", Label(TestCaseLabelsMap[DifferentBackupStatusValidation]...), func() {
```

Comment:
How about we change the name of the test case to `BackupStateTransitionForScheduledBackups`?

Reply:
Addressed.